Energy Web/energy-web-mainnet-starter/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "energy-web-starter",
```

Comment:
How does this work with testnets? Should this include the network type in it?
E.g. `energy-web-testnet-starter`

Reply:
The name should probably match the name of the directory

I suggest we format this network name like so
```
const name = `${name.split(' ').toLower().join('-')}-${environment.toLower() === 'mainnet' ? '' : environment + '-'}starter`;
```

And then use it in the directory name and in all these other names @bgodlin ?

Reply:
In this case the environment would be named differently right? So this would incorporate it?